resources/flakey-github-comment-markdown.template

```diff
@@ -0,0 +1,17 @@
+## :bug: Flaky test report
```

Comment:
An example of the output from the UTs:

![image](https://user-images.githubusercontent.com/2871786/95224017-21792680-07f2-11eb-8600-e818dbd2e0a8.png)